etn-ccis/blui-react-component-library
# BLUI-3902 re-usable playground component

Fixes BLUI-3902.



#### Changes proposed in this Pull Request:

-  Isolate playground component and refactor as a re-usable component

#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- Uncomment the code in the `__configuration__/navigationMenu/navigation/tsx` in dev docs to add the test-playground routes. 
- Poke around and see how it all works.
- For now, all of the code for this component can be found in the `components/playground` folder, including the example screens and configs
